packages/libs/error-stack/src/lib.rs

```diff
@@ -399,25 +403,26 @@
 //!  Feature   | Description                                                    | implies | default
 //! -----------|----------------------------------------------------------------|---------|--------
 //!  `std`     | Enables support for [`Error`] and, on nightly, [`Backtrace`]   |         | enabled
-//!  `hooks`   |Enables the usage of [`set_display_hook`] and [`set_debug_hook`]| `std`   | disabled
 //! `spantrace`| Enables the capturing of [`SpanTrace`]s                        |         | disabled
 //!  `futures` | Provides a [`FutureExt`] adaptor                               |         | disabled
 //! `small`    | Enable optimizations for the memory footprint of [`Report`]    |         | enabled
 //!  `anyhow`  | Provides conversion from [`anyhow::Error`] to [`Report`]       |         | disabled
 //!   `eyre`   | Provides conversion from [`eyre::Report`] to [`Report`]        |         | disabled
+//!   `glyph`  | Provide color and use of box drawing characters in [`Debug`] output |    | enabled
 //!
 //! [`set_display_hook`]: Report::set_display_hook
 //! [`set_debug_hook`]: Report::set_debug_hook
 //!
 //! [`Error`]: std::error::Error
 //! [`Error::provide`]: std::error::Error::provide
 //! [`Backtrace`]: std::backtrace::Backtrace
-//! [`Display`]: std::fmt::Display
+//! [`Display`]: core::fmt::Display
+//! [`Debug`]: core::fmt::Debug
 //! [`SpanTrace`]: tracing_error::SpanTrace
 //! [`smallvec`]: https://docs.rs/smallvec
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(nightly, feature(provide_any))]
-#![cfg_attr(all(doc, nightly), feature(doc_auto_cfg))]
+#![cfg_attr(all(doc, nightly), feature(doc_auto_cfg, doc_cfg))]
```

Comment:
Out of curiosity: Isn't `doc_cfg` available on stable? Doesn't matter because we only run the docs on `nightly`.
I know you experimented around with this, is this still needed, though?

Reply:
Yep, `doc_cfg` is still unstable. There have been some efforts for stabilization back in 2020, but it seems to be mostly "stuck": https://doc.rust-lang.org/unstable-book/language-features/doc-cfg.html